src/app/components/properties-panel/properties-panel.component.html

```diff
@@ -1,16 +1,62 @@
 <div class="properties-container">
-    <p class="properties-header">Properties Panel</p>
-    <p>stroke width</p>
-    <mat-slider aria-label="units" min="1" max="5" step="0.5" value="1.5"></mat-slider>
-    <p>
-        Stroke
-        <input class="color-container-stroke" type="color" name="strokeColor" id="strokeColor" />
-    </p>
-    <p>
-        Fill
-        <input class="color-container-fill" type="color" name="fillColor" id="strokeColor" />
-    </p>
-    <p>
-        Angle <span><input type="text" /></span>
-    </p>
-</div>
\ No newline at end of file
+  <p class="properties-header">Properties Panel</p>
+  <div>
+    <label>Stroke Width</label>
+    <mat-slider
+      aria-label="units"
+      min="1"
+      max="5"
+      step="0.5"
+      #strokeWidth
+      [disabled]="isDisabled"
+      [(ngModel)]="objProperties.strokeWidth"
+      (change)="setProperties()"
+    ></mat-slider>
+  </div>
+
+  <div>
+    <label>Stroke</label>
+    <span>
+      <input
+        class="strokeColorContainer"
+        type="color"
+        #strokeColor
+        [(ngModel)]="objProperties.stroke"
+        [disabled]="isDisabled"
+        (change)="setProperties()"
+      />
+    </span>
+  </div>
+
+  <br />
+
+  <div>
+    <label>Fill</label>
+    <span>
+      <input
+        class="fillColorContainer"
+        type="color"
+        #fill
+        [disabled]="isDisabled"
+        [(ngModel)]="objProperties.fill"
+        (change)="setProperties()"
+      />
+    </span>
+  </div>
+  <br />
+  <label> Angle</label>
+  <span
+    ><input
+      type="number"
+      #angle
+      [disabled]="isDisabled"
+      [(ngModel)]="objProperties.angle"
+      (change)="setProperties()"
+  /></span>
```

Comment:
Try to see if you can get the form value updated as soon as the user changes the angle of the object using the control point. Right now, I need to deselect and select the object again to update the value in the form.